xds/internal/xdsclient/transport/loadreport.go

```diff
@@ -78,7 +83,11 @@ func (t *Transport) lrsStopStream() {
 
 	t.lrsCancelStream()
 	t.logger.Infof("Stopping LRS stream")
+
+	// Wait for the runner goroutine to exit, and set the done channel to nil.
+	// The done channel will be recreated when a new stream is created.
 	<-t.lrsRunnerDoneCh
+	t.lrsRunnerDoneCh = nil
```

Comment:
Technically this is unnecessary, right?  Maybe remove it?

Reply:
Yes, technically unnecessary. Just paranoia I guess.